torch/_dynamo/testing.py

```diff
@@ -147,7 +147,7 @@ def debug_dump(name, code: types.CodeType, extra=""):
         )
 
 
-def debug_insert_nops(frame, cache_size, hooks):
+def debug_insert_nops(frame, cache_size, hooks, code_part):
```

Comment:
weird

Reply:
It's a test util, it has to match the contract.